modules/acquia_cms_headless/modules/acquia_cms_headless_robustapi/src/HeadlessRobustApiInstallHandler.php

```diff
@@ -106,7 +105,7 @@ public function createHeadlessConsumer() {
         // consumer secret before it's stored in a hash key.
         $this->messenger->addStatus($this->t('Your Oauth consumer secret has been generated for you and is: <h2>@secret</h2> Please store this value as it cannot be retrieved. Alternatively you can generate a new secret via the <a href="@link">Consumer UI</a>', [
           '@secret' => $secret,
-          '@link' => Url::fromRoute('consumers.admin.config.services'),
+          '@link' => ('config/services/consumer'),
```

Comment:
I was hoping to use a link function here.  such as `URL()`.  Regardless, shouldn't it be `/adminconfig/services/consumer`?

Reply:
I tried this options:
URL::fromURI(internal:...) internal and it still called admin/modules (is that what you mean?)
putting /admin in the link generates a link: admin/admin....

Reply:
Odd.  Okay.  We can leave it as is for now then if it's working.

Reply:
it was a last-ditch effort it seemed weird to me that it worked.